Data/UnitTests/GameEngineTest/PlatformWin/Kraut/cgb-MossyBark01.ezMaterialAsset

```diff
@@ -1,878 +0,0 @@
-HeaderV2
-{
-o
-{
-	Uuid %id{u4{10130857189507589773,5275827593244715358}}
-	s %t{"ezAssetDocumentInfo"}
-	u3 %v{2}
-	s %n{"Header"}
-	p
-	{
-		s %AssetType{"Material"}
-		VarArray %Dependencies
-		{
-			s{"{ d792c892-581f-4de9-b83a-0599b1b48770 }"}
-		}
-		Uuid %DocumentID{u4{10130857189507589773,5275827593244715358}}
-		u4 %Hash{5108684433292054812}
-		VarArray %MetaInfo{}
-		VarArray %Outputs{}
-		VarArray %References
-		{
-			s{"Grey.color"}
-			s{"{ d792c892-581f-4de9-b83a-0599b1b48770 }"}
-		}
-	}
-}
-}
-Objects
-{
-o
-{
-	Uuid %id{u4{12393651040102028521,4891228368794462065}}
-	s %t{"Kraut/Kraut.ezShader"}
-	u3 %v{2}
-	p
-	{
-		s %BLEND_MODE{"BLEND_MODE::BLEND_MODE_OPAQUE"}
-		s %BaseTexture{"Grey.color"}
```

Comment:
This material was copied over 1:1 and then adjusted to not use a texture. Now you get the original asset and that's why the tree appears textured again in the test and therefore fails now.

Reply:
Ah that explains the guid collision, thanks.

Reply:
Ok I've restored it with a new guid and changed the material references fo the tree to the new asset.